src/Numeric/Interval/NonEmpty/Internal.hs

```diff
@@ -77,6 +78,12 @@ import qualified Data.Semigroup
 -- >>> let conservativeExceptNaN sf f xs = forAll (choose (inf xs, sup xs)) $ \x -> isNaN (sf x) || (sf x) `member` (f xs)
 -- >>> let compose2 = fmap . fmap
 -- >>> let commutative op a b = (a `op` b) == (b `op` a)
+--
+--
+-- eta expanded to avoid defaulting with older GHCi
```

Comment:
Another occurrence of this comment, which I think ought to be removed.

Reply:
Oh wait, _that_ doesn't look good: https://github.com/ekmett/intervals/pull/65/checks?check_run_id=1678359057#step:20:39

I wonder if there's some GHC 7.6-specific `ExtendedDefaultRules` bug that we're hitting. In that case, I take back my advice about eta contracting these definitions. Sigh.